index.html

```diff
@@ -165,6 +165,16 @@ <h2 class="title">Form</h2>
           </div>
         </div>
       </div>
+      <div>
```

Comment:
I think this change doesn't belong to this PR, right?

Can we remove it please?

Reply:
Sorry, I removed it from this one.